src/features/PesterTests.ts

```diff
@@ -18,12 +18,12 @@ export class PesterTestsFeature implements IFeature {
         this.command = vscode.commands.registerCommand(
             "PowerShell.RunPesterTestsFromFile",
             () => {
-                this.launchTests(vscode.window.activeTextEditor.document.uri, false);
+                this.launchTests(vscode.window.activeTextEditor.document.uri, false, undefined);
```

Comment:
Yeah, the same thought crossed my mind as I was coding it up.  It can obviously be made to work but feels brittle.  I think we could handle this better - perhaps have a fourth parameter to launch function which indicates the desire is to debug/run the whole file.  Let me take a crack at this tonight.